lambdas/TranscriptionJobStartLambdaFunction/index.js

```diff
@@ -26,8 +26,15 @@ const transcribe = new AWS.TranscribeService({ apiVersion: '2017-10-26' });
 exports.handler = async (event) => {
   console.log(JSON.stringify({ msg: 'State input', input: event }));
 
+  const extension = event.Artifact.Descriptor.Extension;
+
+  // Take your life in your own hands, force a format
+  if (event.Task.MediaFormat) {
+    extension = event.Task.MediaFormat;
+  }
+
   // Only start the job if the artifact type is supported
```

Comment:
This comment should also get updated to reflect that it's not just looking at the artifact type anymore

Reply:
ok

Reply:
updated comment